utils/mode.go

```diff
@@ -0,0 +1,50 @@
+package oyster_utils
+
+import (
+	"github.com/getsentry/raven-go"
+	"github.com/joho/godotenv"
+	"log"
+	"os"
+)
+
+type ModeStatus int
+
+const (
+	ProdMode ModeStatus = iota + 1
+	TestModeDummyTreasure
+	TestModeNoTreasure
+)
+
+var BrokerMode ModeStatus
+
+func init() {
+
+	// Load ENV variables
+	err := godotenv.Load()
+	if err != nil {
+		log.Println("Error loading .env file")
+		raven.CaptureError(err, nil)
+	}
+
+	brokerMode := os.Getenv("MODE")
+
+	SetBrokerMode(brokerMode)
```

Comment:
this is a private method? maybe do: setBrokerMode()

Reply:
Okay I'll make it private for now.  We may have to re-expose it if we decide to do any testing of the different broker modes.